main.py

```diff
@@ -0,0 +1,15 @@
+from typing import Optional
```

Comment:
This line looks extraneous. So let's remove it as well as the `GET` route to `/items/{item_id}` and the code to support it below (lines 13-15).

Reply:
Great, It was just a starting point, I knew there was no purpose for it 👍🏼